app/components/layer-advanced/template.hbs

```diff
@@ -1,5 +1,12 @@
 {{edit-text showSettings=showSettings editMode=editMode layer=layer type="title"}}
 
-<div id="froala-editor"  class="fr-view">
-</div>
+<!-- <div id="froala-editor"  class="fr-view">
+</div> -->
 
+{{#if editMode}}
+	{{quill-editor value=layer.content options=options textChange=(action "updateText")}}
+{{else}}
+<div  class="ql-editor">
```

Comment:
ql-editor class adds padding to the sides, we should remove the side padding by overriding the rule.